dev_requirements.txt

```diff
@@ -17,4 +17,8 @@ pytest-xdist
 pytz
 tox>=3.13
 twine
-wheel
\ No newline at end of file
+wheel
+agate==1.6.3
+requests==2.28.1
+setuptools==63.2.0
+third-party-license-file-generator
```

Comment:
why do we need this? I don't think we used this at all.

Reply:
Yeah, accidentally got pulled in with one of the license generator tools I was using. I will remove it.